### PR TITLE
Upgraded to SuiteTalk 2021.2

### PR DESCRIPTION
**Pull request by SampathDilhan**

### CODE REVIEW

**Comment by SampathDilhan** [comment]
- Upgraded to SuiteTalk 2021.2
- Removed support for Basic Credentials.

File: Celigo.SuiteTalk/src/SuiteTalk/INetSuiteClient.cs [hunk 1]
```diff
@@ -3,11 +3,11 @@
 
 namespace SuiteTalk
 {
-    public partial interface INetSuiteClient : NetSuitePortType, IPassportProvider, IPreferenceProvider
+    public partial interface INetSuiteClient : NetSuitePortType, IPreferenceProvider
     {
 #pragma warning disable IDE1006 // Naming Styles
         ApplicationInfo applicationInfo { get; set; }
-        Passport passport { get; set; }
+        // Passport passport { get; set; }
```
Comment (Jagapathi126) [inline Celigo.SuiteTalk/src/SuiteTalk/INetSuiteClient.cs:10]: Lets completely remove these properties?

File: Celigo.SuiteTalk/src/SuiteTalk/Proivder.types.cs [hunk 2]
```diff
@@ -1,9 +1,9 @@
 ﻿namespace SuiteTalk
 {
-    public interface IPassportProvider
-    {
-        Passport GetPassport();
-    }
+    // public interface IPassportProvider
```
Comment (Jagapathi126) [inline Celigo.SuiteTalk/src/SuiteTalk/Proivder.types.cs:3]: Same here

**Review by Jagapathi126 — COMMENTED** [review]
Continuing the review

**Review by sameera — APPROVED** [review]

**Pull request merged**